apps/opik-documentation/documentation/fern/docs/evaluation/metrics/task_span_metrics.mdx

```diff
@@ -0,0 +1,364 @@
+---
+description: Learn how to create task span metrics for evaluating the detailed execution information of your LLM tasks
+toc_max_heading_level: 4
+---
+
+# Task Span Metrics
+
+Task span metrics are a powerful type of evaluation metric in Opik that can analyze the detailed execution information of your LLM tasks. Unlike traditional metrics that only evaluate input-output pairs, task span metrics have access to the complete execution context, including intermediate steps, metadata, timing information, and hierarchical structure.
```

Comment:
Let's emphasize that these metrics only work as a part of evaluate function (because without it the users simply will not be able to get SpanModel instance).

In addition, let's also mention, that only the spans created with `@track` decorator and our native integrations will be presented in the SpanModel tree.